src/packages/solid/.storybook/preview.tsx

```diff
@@ -25,8 +25,8 @@ const RenderOptions = {
   coreExtensionModule: coreExtensionModuleUrl,
   threadXCoreWorkerUrl: undefined,
   rootId: 'storybook-root' as any,
-  appWidth: 900,
-  appHeight: 700,
+  appWidth: 1920,
```

Comment:
hmmm

Reply:
maybe lets just peg this at 1280x720 and stop messing with it

Reply:
okay, i had just changed it so the full keyboard could show and it matched screenW